framework/src/modules/chain/transport/schemas.js

```diff
@@ -162,20 +156,15 @@ module.exports = {
 		},
 	},
 	transactionsRequest: {
```

Comment:
The schema names are not following any pattern at all. Can we have one? Otherwise, let's call this new one `transactionIDsRequest` at least.

Reply:
I will update all of them in another issue